core/src/main/java/io/kestra/core/runners/Worker.java

```diff
@@ -73,6 +73,7 @@ public class Worker implements Runnable, AutoCloseable {
     @Getter
     private final Map<Long, AtomicInteger> metricRunningCount = new ConcurrentHashMap<>();
 
+    @Getter
```

Comment:
Can you add `@VisibleForTesting` to know that it's only used for testing purpose?

Reply:
Done